reco_utils/dataset/covid_utils.py

```diff
@@ -107,38 +103,36 @@ def clean_dataframe(df):
     return df
 
 
-def retrieve_text(entry, blob_service, container_name):
+def retrieve_text(
+        entry, 
+        container_name,
+        azure_storage_account_name="azureopendatastorage",
+        azure_storage_sas_token="sv=2019-02-02&ss=bfqt&srt=sco&sp=rlcup&se=2025-04-14T00:21:16Z&st=2020-04-13T16:21:16Z&spr=https&sig=JgwLYbdGruHxRYTpr5dxfJqobKbhGap8WUtKFadcivQ%3D",
```

Comment:
yeah i don't like it either. we can remove it and add instructions in the notebook on where to find the token from azure open datasets. did you add the covid notebook originally? can you update it to avoid including the sas token here?

Reply:
removed